### ISSUE TITLE
[Windows Terminal - Appearance]: Text is truncated for 'Hide Terminal in the notification area when it is minimized' and 'Display a shield in the title bar when Windows Terminal is running as Administrator' after applying reflow settings.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#20603]
### Windows Terminal version

1.25.1912.0

### Windows build number

28120.2738

### Other Software

**Test Environment:**
OS Version: Windows 11 version 26H1 (OS Build 28120.2738)
App Name: Windows Terminal Preview

### Steps to reproduce

**Pre-requisites for applying reflow:**
>Change the display resolution to '1280 x 1024', set scaling to '100%'.
>Open the application & resize it to minimum.
>Turn on Magnifier at 400% zoom & verify whether we are able to view content in one dimension of '320 px' view port or not.

**Repro Steps:**

1. Launch the 'Windows Terminal' application.
2. Press 'Ctrl + ,' keys to open 'Settings'.
3. Navigate to the 'Appearance' section.
4. Scroll down to the settings options.
5. Observe the 'Hide Terminal in the notification area when it is minimized' setting.
6. Observe the 'Display a shield in the title bar when Windows Terminal is running as Administrator' setting.

**User Experience:**
Users who require reflow may be unable to read the complete description of these settings because portions of the text are truncated.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/reflow.html

**Attachment:**
https://onedrive.cloud.microsoft/:f:/a@rc32u9qu/S/IgBxFhPLSyM0TbF5i1HWnulEAeGfFQTYnSQsCX9mXWSthN0?e=yh5ZHu

### Expected Behavior

The complete text for all settings should remain fully visible after applying the reflow settings. Text should wrap and reflow within the available space without being truncated, clipped, or hidden.

### Actual Behavior

After applying the reflow settings, the text for the following settings is partially truncated and not fully visible:

- 'Hide Terminal in the notification area when it is minimized'
- 'Display a shield in the title bar when Windows Terminal is running as Administrator'
